package.json

```diff
@@ -23,8 +30,21 @@
     "tslint-plugin-prettier": "^2.3.0",
     "typescript": "^3.8.3"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn workspaces run lint-staged"
+    }
+  },
   "dependencies": {
-    "@stitches/core": "file:packages/css",
-    "@stitches/react": "file:packages/styled"
+    "@babel/core": "^7.11.5",
+    "@babel/preset-env": "^7.11.5",
+    "@babel/preset-typescript": "^7.10.4",
+    "@preconstruct/cli": "^1.1.27",
+    "react": "^16.13.1"
+  },
+  "preconstruct": {
```

Comment:
whaqts preconstruct?

Reply:
https://preconstruct.tools/
it makes life a lot easier when working with libraries in monorepos